projects/angular/src/data/datagrid/datagrid.ts

```diff
@@ -347,6 +349,10 @@ export class ClrDatagrid<T = any> implements AfterContentInit, AfterViewInit, On
             this._calculationRows.insert(row._view);
           });
         }
+      }),
+      this.visiblityChange.subscribe(visibility => {
```

Comment:
Do we need a subject and a subscription for this? Why not just a `setVisibility` method. I think this subscribe callback happens synchronously upon `next`, so this feels like unnecessary indirection.